l10n_ar_ux/controllers/portal.py

```diff
@@ -48,8 +48,21 @@ def account(self, redirect=None, **post):
         uid = request.session.uid
         partner = request.env['res.users'].browse(uid).partner_id if uid else request.env['res.partner']
         partner = partner.with_context(show_address=1).sudo()
+        #recomputamos el commercial partner porque al editar el fomrulario perdemos ese dato
+        partner._compute_commercial_partner()
+        #Al guardar por primera vez los datos del contacto, se pierden la responsabilidad y el tipo de identificacion
+        if not partner.l10n_ar_afip_responsibility_type_id and post.get('l10n_ar_afip_responsibility_type_id'):
+            resp = int(post.get('l10n_ar_afip_responsibility_type_id'))
+            partner.l10n_ar_afip_responsibility_type_id = request.env['l10n_ar.afip.responsibility.type'].browse(resp)
+        if not partner.l10n_latam_identification_type_id and post.get('l10n_latam_identification_type_id'):
+            ident = int(post.get('l10n_latam_identification_type_id'))
+            partner.l10n_latam_identification_type_id = request.env['l10n_latam.identification.type'].browse(ident)
```

Comment:
1. check if the elements came in **post
2. verificar el if not error or if error (check remove it temporally to see what it happens.
3. delete set of values on database partner.field = value